spec/api/verifications_spec.rb

```diff
@@ -40,6 +40,37 @@
           run_test!
         end
       end
+
+      put('create_or_update verification') do
```

Comment:
```suggestion
      put('create or update verification') do
```

this is a doc comment